code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -21,6 +21,9 @@
  * is handled by the hotspot itself, specifically perform_exposure().
  */
 /turf/open/hotspot_expose(exposed_temperature, exposed_volume, soh)
+	if(exposed_temperature < TCMB)
+		CRASH("[src].hotspot_expose() called with exposed_temperature < [TCMB]")
+		exposed_temperature = TCMB
```

Comment:
```suggestion
	if(exposed_temperature < TCMB)
		exposed_temperature = TCMB
		CRASH("[src].hotspot_expose() called with exposed_temperature < [TCMB]")
```
Any code logic after `CRASH` will not execute, so you need to rearrange the code.